examples/bri-3/test/zkp/merkleTree.test.ts

```diff
@@ -0,0 +1,42 @@
+import { MerkleTree } from "../../src/bri/zkp/components/merkleTree/merkleTree";
+
+describe("Create a Merkle Tree and return a unique root", () => {
```

Comment:
can we add tests for leaves too? i can help with adding those tests so i can also learn about library a bit if you don't mind @biscuitdey ?

Reply:
+1

Reply:
+1